backend/apps/accounts/migrations/0001_initial.py

```diff
@@ -1,86 +0,0 @@
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-
-    initial = True
-
-    dependencies = [
-        ("auth", "0012_alter_user_first_name_max_length"),
-    ]
-
-    operations = [
-        migrations.CreateModel(
-            name="User",
-            fields=[
-                (
-                    "id",
-                    models.BigAutoField(
-                        auto_created=True,
-                        primary_key=True,
-                        serialize=False,
-                        verbose_name="ID",
-                    ),
-                ),
-                ("password", models.CharField(max_length=128, verbose_name="password")),
-                (
-                    "last_login",
-                    models.DateTimeField(
-                        blank=True, null=True, verbose_name="last login"
-                    ),
-                ),
-                (
-                    "is_superuser",
-                    models.BooleanField(
-                        default=False,
-                        help_text="Designates that this user has all permissions without explicitly assigning them.",
-                        verbose_name="superuser status",
-                    ),
-                ),
-                (
-                    "email",
-                    models.EmailField(
-                        db_index=True,
-                        max_length=254,
-                        unique=True,
-                        verbose_name="Email address",
-                    ),
-                ),
-                (
-                    "is_active",
-                    models.BooleanField(default=False, verbose_name="Is active"),
-                ),
-                (
-                    "is_admin",
-                    models.BooleanField(default=False, verbose_name="Has admin rights"),
-                ),
-                (
-                    "groups",
-                    models.ManyToManyField(
-                        blank=True,
-                        help_text="The groups this user belongs to. A user will get all permissions granted to each of their groups.",
-                        related_name="user_set",
-                        related_query_name="user",
-                        to="auth.group",
-                        verbose_name="groups",
-                    ),
-                ),
-                (
-                    "user_permissions",
-                    models.ManyToManyField(
-                        blank=True,
-                        help_text="Specific permissions for this user.",
-                        related_name="user_set",
-                        related_query_name="user",
-                        to="auth.permission",
-                        verbose_name="user permissions",
-                    ),
-                ),
-            ],
-            options={
-                "abstract": False,
-            },
-        ),
-    ]
```

Comment:
Why did you delete migrations from the app you're not even working with??
Btw, you're not allowed to just delete migrations the other teammates already have.

Reply:
> Why did you delete migrations from the app you're not even working with?? Btw, you're not allowed to just delete migrations the other teammates already have.

Because precommit don't miss it. 
.gitignor should not skip migrations.

Reply:
First of all, please revert changes and return those migrations back.
Then find out where the problem came from (which tool is exactly removing the migration files), because we had excluded the migrations inside the `pre-commit-config.yaml` file.
And .gitignore is not staging migrations, what's actually right.